### DISCUSSION TITLE
arm64 docker missing sandbox.yaml map

### DISCUSSION THREAD

**Discussion by Travis-Hainsworth** [prl-mushr/mushr#92] [Software]
First off, I love what you all are doing! Thank you for making this car and ros environment!

FYI - the mushr/mushr:aarch64 docker image is missing sandbox.yaml in catkin_ws/src/mushr_sim/maps

<img width="620" alt="image" src="https://user-images.githubusercontent.com/33101501/201239708-57931608-4ad2-448a-a458-a629f4e27357.png">

which is the default map in the teleop.launch file

**Comment by MarkusSchiffer** [comment]
Hello Travis!
Thanks for checking out MuSHR. I just pushed a fix to the noetic branch of mushr_sim adding the sandbox map. You should get it if you just pull the latest!

**Comment by Travis-Hainsworth** [comment]
I'm actually selling my M1 MAC, so will be abandoning the arm64 docker. I'll keep you posted on how x86 usage goes though! :)
